src/features/wallet/root/WalletScreen.tsx

```diff
@@ -12,15 +10,7 @@ const WalletScreen = () => {
         <WalletView />
       </Box>
 
-      {/* For some reason having a BottomSheet (which is in ActivityDetails)
-      contained in a <Portal/> breaks the header drag handler on Android.
-      Conditionally removing the portal on Android for now  */}
-      {Platform.OS === 'android' && <ActivityDetails />}
```

Comment:
@matthewcarlreetz can you check that this part works ok on Android?

Reply:
yep, the code is looking good. I will do some thorough device testing next...

Reply:
It looks like this still is an issue. The drag handle on android doesn't work for the detail view. Clicking the background to close does work though.